pyscripts/horton_create_identities.py

```diff
@@ -11,149 +11,96 @@
-            deployment_name += '_' + id_prefix
             for azure_device in identity_json:
-                az_device_id = identity_json[azure_device]
-                objectType = self.get_json_value(az_device_id, 'objectType')
-                objectName = self.get_json_value(az_device_id, 'objectName')
+                device_json = identity_json[azure_device]
+                objectType = device_json['objectType']
+                objectName = device_json['objectName']
+                device_id = "{}_{}".format(id_prefix, objectName)
                 if objectType == "iothub_device":
-                    device_id = "{}_{}".format(id_prefix, objectName)
-                    dev_object = self.populate_device_object(az_device_id, objectName, device_id)
-                    dev_object.connectionString = self.create_iot_device(hub_connect_string, device_id)
-                    dev_object.deviceId = device_id
-                    child_modules = []
-                    if(self.node_has_children(identity_json, azure_device, 'modules')):
-                        modules = identity_json[azure_device]['modules']
+                    device_json['deviceId'] = device_id
+                    device_json['connectionString'] = self.create_iot_device(hub_connect_string, device_id)
+                    device_count += 1
+                    if 'modules' in device_json:
+                        modules = device_json['modules']
                         for module in modules:
-                            module_obj = self.populate_device_object(modules[module], module, device_id)
-                            module_obj.deviceId = device_id
-                            module_obj.moduleId = module
-                            module_obj.connectionString = self.create_device_module(hub_connect_string, device_id, module)
-                            child_modules.append(module_obj)
+                            module_json = modules[module]
+                            module_json['moduleId']  = module
+                            module_json['deviceId']  = device_id
+                            module_json['connectionString']  = self.create_device_module(hub_connect_string, device_id, module)
+                            deployment_json['identities'][azure_device][module] = module_json
                             module_count += 1
+                elif objectType in ["iothub_service", "iothub_registry"]:
+                    print("creating service {}".format(device_id))
+                    device_json['deviceId'] = device_id
+                    device_json['connectionString'] = hub_connect_string
+                    device_count += 1
+                deployment_json['identities'][azure_device] = device_json
+        except:
+            print(Fore.RED + "Exception Processing HortonManifest: " + save_manifest_file + Fore.RESET, file=sys.stderr)
```

Comment:
> + Fore.RESET [](start = 89, length = 13)

if you init with autoreset=True, you can remove every single + Fore.RESET in the review because that causes colorama to reset automatically at the end of every print call.